pkg/errors/armerrors_test.go

```diff
@@ -0,0 +1,146 @@
+package errors
+
+
+import (
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"testing"
+
+    "bytes"
+    "io"
+    "net/http"
+
+)
+
+func TestIsNotFoundErr(t *testing.T) {
```

Comment:
Is there not a library func we could use to asset/expect the values rather than the if statements?